src/main/thrift/parquet.thrift

```diff
@@ -1115,6 +1189,34 @@ union EncryptionAlgorithm {
   2: AesGcmCtrV1 AES_GCM_CTR_V1
 }
 
+/**
+ * Embedded metadata page.
+ * 
+ * A metadata page is a data page used to store metadata about
+ * the data stored in the file. This is a key feature of PAR3
+ * footers which allow for deferred decoding of metadata.
```

Comment:
So the page will have a PageHeader/DataPageHeader at the top? Will nulls or repetition be allowed, i.e. do we need definition and repetition level data? If not, then should we define a new page type instead so we don't have to encode unused level encoding types? Then we could also drop the language below about not writing statistics.

Reply:
This is also a reasonable approach to define a new page type.  I was leaving this open in case in the future we want nulls.  Whether nulls are allowed and exact structure is dictated by its use on the field and also to minimize spec changes in this draft.  The nice thing about this approach is it can work transparently if/when a new page type is added.